src/experiments/playground/src/pages/default.astro

```diff
@@ -1,7 +1,660 @@
 ---
-import BaseLayout from '../layouts/BaseLayout.astro'
+import 'tailwindcss/tailwind.css';
 ---
-
-<BaseLayout>
-  <button class="btn">Hello daisyUI</button>
-</BaseLayout>
\ No newline at end of file
+<html>
+  <head>
+    <title>Playground</title>
+  </head>
+  <body class="flex flex-col justify-center items-center min-h-[100vh]">
```

Comment:
It works, but why don't you use

```html
<BaseLayout>
  <div class="flex flex-col justify-center items-center min-h-[100vh]">
    <!-- content -->
  </div>
</BaseLayout>
```

Just curious if building HTML skeleton from the ground up will make any difference.